client-src/src/pages/inq-details/inq-details.html

```diff
@@ -13,7 +13,15 @@
 <ion-content>
   <form [formGroup]="inqForm" (ngSubmit)="logForm()">
     <ion-grid>
-      <ion-item-divider color="light">Personal Details<span class="inq-status" *ngIf="currentInq">Status: {{currentInq.data.inquiryStatus}}</span></ion-item-divider>
+      <ion-item-divider color="light">Personal Details
+        <span class="inq-status" *ngIf="currentInq">
+          Status:
+          <select (change)="changeInqStatus($event)">
+            <option [selected]=isInqOpen(currentInq.data.inquiryStatus) value="open">Open</option>
+            <option [selected]=!isInqOpen(currentInq.data.inquiryStatus) value="close">Closed</option>
+          </select>
+        </span>
+      </ion-item-divider>
```

Comment:
share updated screen shot

Reply:
![](https://image.prntscr.com/image/i3H1QOezQRafug2rGygd1w.png)

Reply:
Make this a little wider. Also include closing_status, closing_substatus

Reply:
resolved in commit 95cfd46